### DISCUSSION TITLE
1.0.0-RC5

### DISCUSSION THREAD

**Discussion by Wirone** [Codito-Dev/rector-money#8] [Announcements]
## What's Changed

* Bump to Rector ^1.0 by @samsonasik in https://github.com/Codito-Dev/rector-money/pull/6
* Drop dependency to `webmozart/assert` by @Wirone

**Full Changelog**: https://github.com/Codito-Dev/rector-money/compare/1.0.0-RC4...1.0.0-RC5

<hr /><em>This discussion was created from the release <a href='https://github.com/Codito-Dev/rector-money/releases/tag/1.0.0-RC5'>1.0.0-RC5</a>.</em>

**Comment by Wirone** [comment]
I believe it's about time to created stable `1.0.0` tag, do you agree 😁?

**Reply by samsonasik** [reply]
Sounds good to me 👍
